Makefile

```diff
@@ -0,0 +1,42 @@
+IMAGE_REPO ?= quay.io/nmstate
+IMAGE_TAG ?= latest
+MANAGER_IMAGE ?= kubernetes-nmstate-manager
+
```

Comment:
We can compose here the MANAGER_IMAGE_URL to reuse it 
```make
MANAGER_IMAGE_URL ?= $(IMAGE_REPO)/$(MANAGER_IMAGE):$(MANAGER_IMAGE_TAG)
```

Reply:
it's not an URL really (misses the protocol prefix), called it MANAGER_IMAGE_FULL_NAME